explorer/src/queries/search.ts

```diff
@@ -7,6 +7,12 @@ export interface ISearchParams {
   limit?: number
 }
 
+const normalizeSearchToken = (id: string): string => {
+  if (id && id.substr(0, 2) !== '0x' && (id.length === 64 || id.length === 40))
```

Comment:
How do you feel about extracting the magic numbers `64` & `40` into meaningful const variables?

Reply:
I agree. Do you like the introduced names?

Reply:
Looking good. Just a couple of linting errors

Reply:
Need to make a precommit hook or else they're getting out of hand.
@rupurt Don't we want to support fuzzy searching? In my opinion a basic `like %%` to display results for substring queries wouldn't be bad

Reply:
We only want to support exact match searching at this point.